app/src/main/java/tech/relaycorp/gateway/background/CheckInternetAccess.kt

```diff
@@ -10,6 +10,6 @@ class CheckInternetAccess
         pingRemoteServer.pingURL(INTERNET_TEST_WEBSITE)
 
     companion object {
-        private const val INTERNET_TEST_WEBSITE = "https://google.com"
+        private const val INTERNET_TEST_WEBSITE = "https://httpstat.us/500"
```

Comment:
Presumably this is the change you made to test this but it isn't needed in the PR?

Reply:
Sorry, didn't looked at the diff. Just pushed forced a new version.